packages/base/src/toolbar/widget.tsx

```diff
@@ -151,6 +149,17 @@ export class ToolbarWidget extends ReactiveToolbar {
 
       this.addItem('New', NewButton);
 
+      this.addItem('separator2', new Separator());
+
+      this.addItem(
+        'identify',
+        new CommandToolbarButton({
+          id: CommandIDs.identify,
+          label: '',
+          commands: options.commands
+        })
+      );
```

Comment:
Thinking about the future, we could have more buttons here for "point identify", "polygon identify", "bbox identify", and "normal mode"?